.changelog/261.breaking.md

```diff
@@ -0,0 +1,8 @@
+bring Rosetta dependencies up to date
+
+We're updating our Rosetta dependencies, including the
+Go SDK, the CLI, and along with those, the Rosetta API
+specifications.
```

Comment:
Can you also mention the actual versions of the new Rosetta Go SDK, Rosetta CLI and the Rosetta API that we now use?

Reply:
ok